app/javascript/react/components/Video_game_folder/Videogame.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+const Videogame = props => {
+  
+  return (
+    <div className="videogames">
+      <ul>
+        <ul>Title: {props.name}</ul>
+      </ul>
```

Comment:
these shouldnt be uls! one of them can be removed and the other can be a li or p or div